mitodl/micromasters
# Fix mitxonline SSO

#### Pre-Flight checklist


#### What's this PR do?
This resolves an issue that was introduce by https://github.com/mitodl/micromasters/pull/5264.  This also cleans up some of the missing app.json entries for mitxonline.  It also renames some of the widely used attributes to be more generic.

#### How should this be manually tested?
1. Follow the existing documentation to configure micromasters OAuth with edx and mitxonline.
2. Set `MITXONLINE_CALLBACK_URL=http://host.docker.internal:18000/` in your .env file.
3. Create or update a program and course in micromasters so that it is currently active.  
4. Make sure the course is set as an mitxonline course.
5. Test the edx oauth configuration by creating a new micromasters user.
6. Enroll your user into the program from step 2.
7. Visit the user's dashboard to verify the mitxonline OAuth configuration.
